STX.REST.RESTFulSense.Clients/Brokers/Https/HttpBroker.cs

```diff
@@ -14,5 +16,10 @@ public HttpBroker(HttpClient httpClient)
         {
```

Comment:
this could be fat arrow with one line, no curly braces

Reply:
Treated